/*
 * C# Program to Demonstrate DivideByZero Exception
 and handling the exeption
 */
using System;
class Program {
    // main method
    static void Main () {
        try {
            int result = 15 / int.Parse ("0");
            Console.WriteLine (result);
        } catch (DivideByZeroException e) {
            Console.Write (e.Message);
            Console.ReadLine ();
        }
    }
}
/* Output
*   Attempted to Divide by Zero
*/
